src/SFA.DAS.QnA.Application/Queries/Sections/GetSequenceSections/GetSequenceSectionsHandler.cs

```diff
@@ -47,7 +48,7 @@ public async Task<HandlerResponse<List<Section>>> Handle(GetSequenceSectionsRequ
         
         private  void RemovePages(Data.Entities.Application application, Section section)
```

Comment:
Extra whitespace

Reply:
not visible in visual studio

Reply:
It is visible for me:

![image](https://user-images.githubusercontent.com/119664159/220089799-c94aac98-07ba-4f65-a948-049b2d10950e.png)

Also visible in Visual Studio Code

Reply:
done